tests/__init__.py

```diff
@@ -0,0 +1,643 @@
+# run tests: pytest -sv --cov-report term-missing --cov=workflow-ephys -p no:warnings
+
+import os
+import pytest
+import pandas as pd
+import pathlib
+import datajoint as dj
+import importlib
+import numpy as np
+
+from workflow_imaging.paths import get_imaging_root_data_dir
+
+
+@pytest.fixture(autouse=True)
+def dj_config():
+    dj.config.load('./dj_local_conf.json')
+    dj.config['safemode'] = False
+    dj.config['custom'] = {
+        'database.prefix': os.environ.get('DATABASE_PREFIX', dj.config['custom']['database.prefix']),
+        'imaging_root_data_dir': os.environ.get('IMAGING_ROOT_DATA_DIR', dj.config['custom']['imaging_root_data_dir'])
+    }
+    return
+
+
+@pytest.fixture
+def pipeline():
+    from workflow_imaging import pipeline
+
+    yield (pipeline.subject, pipeline.lab, pipeline.imaging,
```

Comment:
might be easier/more straightforward to yield a dictionary & use the **kwargs mechanism in client code
e.g: `yield {'subject': pipeline.subject, ...}`, or potentially define this as a dictionary within pipeline itself and then e.g. `yield pipeline.modules`

Reply:
... although, thinking further - this also requires other functions to be aware of the dictionary contents & name accordingly (e.g. no other 'subject' argument, etc) ...